src/components/Tabs/Tabs.jsx

```diff
@@ -20,30 +21,61 @@ const Tabs = ({
         onSelect: onSelectTab,
         tabIndex: index,
         isActive: index === activeTabIndex,
-        vertical: vertical
+        mode
       });
     });
   };
 
+  const getActiveChild = ({ onClick, open }) => {
+    return React.Children.map(children, (child, index) => {
+      if (index === activeTabIndex) {
+        return React.cloneElement(child, {
+          onClick: onClick
+        });
+      }
+    });
+  };
+
   const transitions = useTransition(activeTabIndex, null, {
     from: { opacity: 0 },
     enter: { opacity: 1 },
     leave: { opacity: 0 },
     config: { duration: 350 }
   });
 
+  const dropdownMode = mode === "dropdown";
+  const vertical = mode === "vertical" || dropdownMode;
+
   return (
     <>
-      <ul
-        className={classNames(
-          vertical ? styles.tabsNavVertical : styles.tabsNav,
-          wrap && styles.wrap,
-          className
-        )}
-        style={style}
-        {...props}>
-        {renderChildrenTabs()}
-      </ul>
+      {dropdownMode ? (
```

Comment:
You can avoid some duplicating the `<ul>...</ul>` code here. You can define it before the `return` statement. Something like this:
```jsx
cons Tabs = ({...}) => {
  ...
  const tabs = (
          <ul
            className={classNames(
              vertical ? styles.tabsNavVertical : styles.tabsNav,
              wrap && styles.wrap,
              className
            )}
            style={style}
            {...props}>
            {renderChildrenTabs()}
          </ul>)
  ...
  return (
   <>
    {dropdownMode ? <Dropdown ... />{tabs}</Dropdown> : tabs }
     ...
   </>);
}
```

Reply:
Just to give you some tips on react performance. In the chunk of code I placed above you could wrap it with the [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo) hook from react. This would memoize the value of  `tabs` and only reevaluate the statement if one of its arguments has changed. Something like:
```jsx
const Tabs = ({...}) => {
...
  const tabs = useMemo(() => (
          <ul
            className={classNames(
              vertical ? styles.tabsNavVertical : styles.tabsNav,
              wrap && styles.wrap,
              className
            )}
            style={style}
            {...props}>
            {renderChildrenTabs()}
          </ul>), 
[vertical, wrap, className, props, renderChildrenTabs]);
...
return (...)
}
```
This will cause the tabs value to be computed only if one of the arguments passed to the array (vertical, wrap, className, props, renderChildrenTabs) changes. 
As you may notice the renderChildrenTabs will always have a different value since this function is being redefined every time the component is rendered. To avoid that you could use the [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback):

```jsx
const Tabs = ({...}) => {
  const renderChildrenTabs = useCallback(() => 
     React.Children.map(children, (child, index) => {
      return React.cloneElement(child, {
        onSelect: onSelectTab,
        tabIndex: index,
        isActive: index === activeTabIndex,
        mode
      });
   ), [children, activeTabIndex, mode]);
  ...
  const tabs = useMemo(() => (
          <ul
            className={classNames(
              vertical ? styles.tabsNavVertical : styles.tabsNav,
              wrap && styles.wrap,
              className
            )}
            style={style}
            {...props}>
            {renderChildrenTabs()}
          </ul>), 
[vertical, wrap, className, props, renderChildrenTabs]);
...
return (...)
}
```
> useCallback will return a memoized version of the callback that only changes if one of the dependencies has changed. This is useful when passing callbacks to optimized child components that rely on reference equality to prevent unnecessary renders